tapir/translations/locale/de/LC_MESSAGES/django.po

```diff
@@ -794,6 +794,12 @@ msgstr "Alle Filter entfernen"
 msgid "Register a new incoming payment"
 msgstr "Eingehende Zahlung registrieren"
 
+#: coop/templates/coop/log/create_payment_log_entry.html:2
+#, fuzzy, python-format
+#| msgid "New Payment: %(amount)s € on %(payment_date)s"
```

Comment:
Is that translation working? I thought that as long as there is the `#, fuzzy` they don't show?

Reply:
strange, yes. must have been auto-added and I didn't see it when I was committing